WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -445,6 +470,23 @@ public void onEventMainThread(NotificationEvents.NotificationsChanged event) {
         mTabLayout.checkNoteBadge();
     }
 
+    @SuppressWarnings("unused")
+    public void onEventMainThread(ConnectionChangeReceiver.ConnectionChangeEvent event) {
+        updateConnectionBar(event.isConnected());
+    }
+
+    private void checkConnection() {
+        updateConnectionBar(NetworkUtils.isNetworkAvailable(this));
+    }
+
+    private void updateConnectionBar(boolean isConnected) {
+        if (isConnected && mConnectionBar.getVisibility() == View.VISIBLE) {
+            AniUtils.animateBottomBar(mConnectionBar, false);
```

Comment:
> I could prevent this by having the connection bar overlay the view pager, but that means the bottom portion of each tab is obscured, which I think is a much worse problem.

That's actually the first thing I verified during the review, would not be good to obstruct the view. Good reason for not implementing the animation :+1:

:shipit: